app/markets/ActivityTable.tsx

```diff
@@ -306,34 +301,47 @@ const LiquidityEventRow = ({ transaction, isAIAgent }: LiquidityEventRowProps) =
             className="h-[14px] w-[14px]"
           />
         </div>
-      </TableCell>
-      <TableCell
-        className="text-nowrap pr-4 text-right text-xs text-text-low-em"
+        <div className="flex items-center space-x-1">
```

Comment:
There is extra space here with `space-x-1`, let's use html whitespace {" "} if possible.